docs/source/terminology.rst

```diff
@@ -2,49 +2,100 @@
+uninitialized ensembles as a baseline for how important (reoccurring) initializations
+are for lending predictability to the system. Some modeling centers (such as NCAR)
+provide a dynamical uninitialized ensemble (the CESM Large Ensemble) along with their
+initialized prediction system (the CESM Decadal Prediction Large Ensemble). If this
+isn't available, one can approximate the unintiailized response by bootstrapping a
+control simulation.
+
+Forecast Assessment
+###################
+
+*Accuracy*: The average degree of correspondence between individual pairs of forecasts
+and observations [Murphy1988]_; [Jolliffe2011]_. Examples include Mean Absolute Error
+(MAE) and Mean Square Error (MSE). See `metrics <metrics.html>`_.
+
+*Association*: The overall strength of the relationship between individual pairs of
+forecasts and observations [Jolliffe2011]_. The primary measure of association is the
+Anomaly Correlation Coefficient (ACC), which can be measured using the Pearson
+product-moment correlation or Spearman's Rank correlation. See
+`metrics <metrics.html>`_.
+
+*(Potential) Predictability*: This characterizes the "ability to be predicted"
+rather than the current "ability to predict." One acquires this by computing a metric
+(like the anomaly correlation coefficient (ACC)) between the prediction ensemble and a
+verification member (in a perfect-model setup) or the reconstruction that initialized
+it (in a hindcast setup) [Meehl2013]_.
+
+*(Prediction) Skill*: Skill assesses the ability of the forecasting system to predict
+the real world, i.e. observations. It must be compared to some "standard of reference"
+to truly be considered skill, such as climatology of persistence [Murphy1988]_.
+
+*Proper Scoring Rule*: A skill score that is defined so that forecasters are rewarded
+with the best expected scores if their forecasts correspond with their judgements
+(rather than e.g. the climatological average). Since forecaster's judements contain
+an element of uncertainty, this concept only applies to probabilstic forecasts
+[Jolliffe2011]_.
+
+*Skill Score*: The most generic skill score can be defined as the following
+[Murphy1988]_:
+
+.. math::
+    S = \\frac{A_{f} - A_{r}}{A_{p} - A_{r}},
+
+where :math:`A_{f}`, :math:`A_{p}`, and :math:`A_{r}` represent the accuracy of the
+forecast being assessed, the accuracy of a perfect forecast, and the accuracy of the
+reference forecast (e.g. persistence), respectively [Murphy1985]_. Here, :math:`S`
+represents the improvement in accuracy of the forecasts over the reference forecasts
+relative to the total possible improvement in accuracy. They are typically designed to
+take a value of 1 for a perfect forecast and 0 for equivelant to the reference
+forecast [Jolliffe2011]_.
+
+*Strictly Proper Scoring Rule*: A skill score where the best scores are obtained if and
+only if the forecasts correspond with the forecaster's judgement [Jolliffe2011]_.
```

Comment:
I dont understand this sentence. what does this mean in practise?

Reply:
I'm not sure. I took this from Jolliffe. I'm going to delete it since I got rid of the stricly proper attribute since we aren't currently using it.